chatbot-ui/values.yaml

```diff
@@ -7,7 +7,9 @@ labels:
 
 name: chatbot-ui
 
-image: 'quay.io/redhat-composer-ai/composer-ui:v0.2.3'
+image:
+  repo: "quay.io/redhat-composer-ai/composer-ui"
+  ref: "v0.2.3"
```

Comment:
Let's rename this to `tag` instead of `ref` to be more consistent with helm defaults